### ISSUE TITLE
AttributeError: EOS is not connected to BOS

### ISSUE THREAD

**Issue by sig-miyamoto** [WorksApplications/SudachiPy#96]
阿Qu
With SudachiPy v0.3.13 + Python 3.6.5 + MacOS 10.14.6, above text causes 'AttributeError: EOS is not connected to BOS'.
(SudachiPy v0.3.12 works correctly.)

**Comment by sorami** [comment]
Hi, thank you for reporting!

Is it really working with `v0.3.12`? Do you have a command and the resulting output for that?

This is a known issue in the original Java Sudachi as well; https://github.com/WorksApplications/Sudachi/issues/45

We understand what is going on in this case, however, we currently do not have a smart idea to solve it without compromising the speed.

Currently, the Java version has an "ignoreError" option for CLI, to just try...catch the error. This is not an elegant solution but that's what we have so far. So if you use SudachiPy as a library, you can also write a `try` .. `except` block to catch the AttributeError for this case.

With the current dictionary, we only expect this to happen when  "阿Q" is followed by an alphabet.

**Comment by sig-miyamoto** [comment]
Thank you for your response.
I think to write a try .. except block is a good solution!

But for your information, v0.3.12 works like ...
```
Python 3.6.5 (default, Apr 19 2018, 11:23:53) 
[GCC 4.2.1 Compatible Apple LLVM 9.1.0 (clang-902.0.39.1)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> from sudachipy import tokenizer
>>> from sudachipy import dictionary
>>> sdc = dictionary.Dictionary().create()
>>> tokenize_mode = tokenizer.Tokenizer.SplitMode.A
>>> m = sdc.tokenize('阿Qu',tokenize_mode)
>>> m[0].surface()
'阿Q'
>>> m[1].surface()
'u'
```

**Comment by izziiyt** [comment]
I know 0.3.12 works.
This bug emerged because I solved another bug #82

**Comment by sorami** [comment]
The problem is finally resolved in the Java version; [Release Sudachi version 0.3.1 · WorksApplications/Sudachi](https://github.com/WorksApplications/Sudachi/releases/tag/v0.3.1)

I am trying to do the same for SudachiPy; #109

**Comment by sorami** [comment]
Finally fixed! Released as `v0.4.2`;
[Release v0.4.2: Bug fix · WorksApplications/SudachiPy](https://github.com/WorksApplications/SudachiPy/releases/tag/v0.4.2)

Please update SudachiPy as follows;

```
$ pip install -U sudachipy
$ pip show sudachipy
Name: SudachiPy
Version: 0.4.2
...
```

Now you should be able to process without the error;

```
$ echo 阿Qu | sudachipy -a
阿      名詞,普通名詞,一般,*,*,*        阿      阿              -1      (OOV)
Qu      名詞,普通名詞,一般,*,*,*        qu      qu              -1      (OOV)
EOS

```
